src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.domain;
+
+import java.util.List;
+import java.util.Random;
+
+public class Cars {
+
+    public static final int RANDOM_BOUND = 10;
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public boolean isEmpty() {
+        return cars == null || cars.isEmpty();
+    }
+
+    public void race(Random random) {
```

Comment:
`Random` 을 직접 파라메터로 받는것보다는 좀 더 정교한 설계를 시도해보시면 좋겠습니다.
지금 필요한게 `Random` 그 자체일까요, 혹은 난수를 발생시키는 전략이나 객체일까요?

Reply:
직접 사용하는 방식보다는

인터페이스나 wrapper클래스로 감싸는 방식을 말씀하시는걸까요?

Reply:
아래 처럼 전략 패턴을 사용하여 주입하도록 변경했습니다

```suggestion
public class Cars {

    private final List<Car> cars = new ArrayList<>();
    private final MovingStrategy strategy; // 자동차 움직임 전략

    public Cars(List<Car> cars, MovingStrategy strategy) {
        this.strategy = strategy;
        this.cars.addAll(cars);
    }

    public boolean isEmpty() {
        return cars.isEmpty();
    }

    public void race() {
        for (Car car : cars) {
            car.move(strategy);
        }
    }
```